tests/test_parsers.py

```diff
@@ -75,12 +80,26 @@ def test_gelfparser_parse_partially_invalid_file():
                 '{"short_message": {"username": "This seems valid too."}}\n',
             ]
         )
-        file.seek(0)
+        with caplog.at_level(logging.WARNING):
+            file.seek(0)
+            parser = GELFParser()
+            events = list(parser.parse(file))
+            parser.parse(file)
 
-        parser = GELFParser()
-
-        events = list(parser.parse(file))
+        file.seek(0)
+        assert (
+            "Input event '{ This is not valid json and raises json.decoder.JSONDecodeError\n' "
+            "is not a valid JSON string! It will be ignored."
+        ) in caplog.text
+        assert (
+            "Input event '{}\n' doesn't comply with GELF format! It will be ignored."
+            in caplog.text
+        )
+        assert (
+            "Input event '[]\n' is not a valid JSON string! It will be ignored."
+            in caplog.text
+        )
```

Comment:
Do you have the error message when the log level is `debug`? It should also be tested :wink:

Reply:
That's right, I only checked for `warning`...